### ISSUE TITLE
Bug in ordering gear

### ISSUE THREAD

**Issue by Jetsurf** [Jetsurf/jet-bot#118]
```Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/discord/client.py", line 317, in _run_event
    await self.on_error(event_name, *args, **kwargs)
  File "/home/discordbot/discordbot/bot.py", line 231, in on_error
    raise exc[1]
  File "/usr/local/lib/python3.7/site-packages/discord/client.py", line 312, in _run_event
    await coro(*args, **kwargs)
  File "/home/discordbot/discordbot/bot.py", line 411, in on_message
    await nsoHandler.orderGear(message, args=args)
  File "./modules/nsohandler.py", line 559, in orderGear
    embed = self.makeGearEmbed(ordered, message.author.name + ", you already have an item on order!", "Respond with 'yes' to replace your order, 'no' to cancel")
  File "./modules/nsohandler.py", line 489, in makeGearEmbed
    embed.set_thumbnail(url='https://splatoon2.ink/assets/splatnet' + gear['gear']['image'])
TypeError: 'NoneType' object is not subscriptable
```

Checking for causes

**Comment by Jetsurf** [comment]
Potentially due to cuttlegear having no common ability, fixed in 0c73b08

**Comment by Jetsurf** [comment]
Issue no longer is appearing. Closing
